store/models.py

```diff
@@ -16,6 +16,14 @@ class Meta:
     def __str__(self):
         return f'{self.title} by {self.author}'
 
+class rating(models.Model):
+    book = models.ForeignKey(Book, related_name='book', on_delete=models.CASCADE)
+    rate = models.FloatField(default=0.0)
+    user = models.ForeignKey(User, related_name='rate', null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
The class name shall start with a capital letter.
The rating shall be an integer - please read proper instructions.
The user should not be null here, and a better option would be to use `on_delete=models.CASCADE`

You could have also used `unique_together` META option here.